recipes-domx/meta-xt-images-vgpu/recipes-core/llvm/llvm/llvm37.patch

```diff
@@ -0,0 +1,805 @@
+diff --git Makefile Makefile
```

Comment:
....and this?

Reply:
All patches mentioned above are copied "as is", no changes from my side.